core/modules/system/system.install

```diff
@@ -203,12 +203,43 @@ function system_requirements($phase) {
     $tasks = new $class();
     $dbinfo = $t('!system version !version', array('!system' => $tasks->name(), '!version' => Database::getConnection()->version()));
     $requirements['database_system'] = array(
-      'title' => $t('MySQL Database'),
+      'title' => $t('MySQL database'),
       'severity' => REQUIREMENT_INFO,
       'value' => $dbinfo,
     );
   }
 
+  // Report cURL version or absence. Don't add severity, since core doesn't
+  // require cURL, but other modules might add their own requirements check if
+  // they need it.
+  // @see simpletest_requirements()
+  if (function_exists('curl_init')) {
+    $curl_version = curl_version();
+    if (is_array($curl_version)) {
+      $version_info = array();
+      if (!empty($curl_version['version'])) {
+        $version_info[] = $t('Version:') . ' ' . $curl_version['version'];
+      }
+      if (!empty($curl_version['ssl_version'])) {
+        $version_info[] = $t('SSL version:') . ' ' . $curl_version['ssl_version'];
+      }
+      if (!empty($curl_version['libssh_version'])) {
+        $version_info[] = $t('libssh version:') . ' ' . $curl_version['libssh_version'];
+      }
+      $curl_status = implode(', ', $version_info);
```

Comment:
Can we please add the ssl and libssh versions in `$requirements['curl']['description]` instead, so that they are shown if someone clicks the "more" link? They would also be easier to read as a bulleted list (`theme('item_list', ...)`).